recipes/album/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "album" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 83117018838599ad49d98d023baa43adf4169d3e1d7a85dd4164fa23ac5e8195
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - album=album.__main__:startup
+    - album-catalog-admin=album.ci.__main__:startup
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - pip
+    - python
+    - pyyaml >=6.0
+    - validators >=0.18.2
+    - git >=2.37
+    - gitpython >=3.1.13
+    - requests >=2.28
+    - pexpect >=4.8.0
+    - album-runner >=0.5.1
+    - jsonschema >=4.0.0
+    - colorlog >=6.6.0
+    - pywin32 >=303  # [win]
```

Comment:
You should replace that package with `pywin32-on-windows`, which is a virtual package for `pywin32` on windows and empty on non windows platforms. This allows you to remove the selector and make this recipe `noarch: python`. See https://conda-forge.org/docs/maintainer/knowledge_base.html#empty-python-packages